actionpack/lib/action_controller/log_subscriber.rb

```diff
@@ -67,6 +67,16 @@ def #{method}(event)
       METHOD
     end
 
+    %w(write_fragment read_fragment).each do |method|
+      class_eval <<-METHOD, __FILE__, __LINE__ + 1
+        def #{method}(event)
+          return unless logger.info? && ActionController::Base.enable_fragment_cache_logging
```

Comment:
Yup, it looks like this will remove messages like:

```ruby
Read fragment views/v1/2914079/v1/2914079/recordings/70182313-20160225015037000000/d0bdf2974e1ef6d31685c3b392ad0b74 (0.6ms)
```

I think we should just let the config work for every one of these methods. The config name communicates as much.